nerdlandbot/commands/GuildData.py

```diff
@@ -263,7 +265,33 @@ async def update_notification_audit(self, list_name: str) -> bool:
         await self.save()
 
         return True
+    
+    async def set_pet(self, pet_name: str, user_id: str) -> str:
+        pets = self.pets
+
+        if len(pets) == 0:
+            pet_id = str(1)
+        else:
+            id_list_strings = list(pets.keys())
+            mapped_list = map(int,id_list_strings)
+            id_list_ints = list(mapped_list)
+            sorted_id_list = sorted(id_list_ints)
+            pet_id = str(sorted_id_list[-1] + 1)
```

Comment:
Rather than this, you should use something like a [UUID](https://docs.python.org/3/library/uuid.html#uuid.uuid4)

Using this strategy is considered bad practice, since an ID could get reused (in your case if the last pet that got added gets removed again).

Reply:
Just so this is documented, we discussed this in PM and decided on the method implemented on commit 70a7e6c (Added unique ID's..).